app/views/admin/loan_questions/_loan_types.slim

```diff
@@ -0,0 +1,31 @@
+= f.input "loan_types" do
+  - if @loan_question.parent
+    / TODO: Use simple form yml to add contextual text for options
+    = f.input_field :override_associations, as: :select
```

Comment:
This field name is correct. However, the label is not correct. @hooverlunch: How can simple form yml based labels be used here?